x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,718 @@
+the old CA certificate.
++
+If you previously used the `elasticsearch-certutil` tool to generate your
+keystore, the alias of the old CA defaults to `ca` and the type of entry is
+`PrivateKeyEntry`.
++
+[source,shell]
+----
+keytool -delete -noprompt -alias ca  -keystore config/elastic-stack-ca.p12 \
+-storepass <password>
+----
++
+[%collapsible%open]
+.Command parameters
+====
+   `-alias`:: Name of the keystore alias for the old CA certificate that you want to remove from your
+   truststore.
+====
+
+[discrete]
+[[transport-layer-newca-whatsnext]]
+==== What's next?
+Well done! You've updated the keystore for the transport layer. You can also
+<<node-certs-different-http,update the keystore for the HTTP layer>> if
+necessary. If you're not updating the keystore for the HTTP layer, then you're
+all set. 
+
+[[node-certs-different-http]]
+==== Generate a new certificate for the HTTP layer
+You can generate certificates for the HTTP layer using your new CA certificate
+and private key. Other components such as {kib} or any of the Elastic language
+clients verify this certificate when they connect to {es}.
+
+NOTE: If your organization has its own CA, you'll need to 
+<<certutil-csr,generate Certificate Signing Requests (CSRs)>>. CSRs contain
+information that your CA uses to generate and sign a security certificate
+instead of using self-signed certificates that the `elasticsearch-certutil` tool
+generates.
+
+Before generating new certificates for the HTTP layer, you need to go to all the clients that connect to {es} (such as {kib}, {beats}, {ls}, and any language
+clients) and configure them to also trust the new CA (`ca.crt`) that you
+generated.
+
+. For *every* client that connects to {es}, import the new `ca.crt` certificate
+into your existing CA truststore for the HTTP layer. This step ensures that your
+cluster trusts the new CA certificate. This example uses the Java `keytool`
+utility to import the certificate into the `http.p12` CA truststore.
++
+[source,shell]
+----
+keytool -importcert -trustcacerts -noprompt -keystore http.p12 \
+-storepass <password>  -alias new-ca -file ca.crt
+----
++
+[%collapsible%open]
+.Command parameters
+====
+   `-keystore`:: Name of the HTTP truststore that you are importing the new CA
+   certificate into. 
+
+   `-storepass`:: Password for the CA truststore. 
+
+   `-alias`:: Name that you want to assign to the new CA certificate entry in
+   the keystore.
+
+   `-file`:: Name of the new CA certificate to import.
+====
+
+. Check that the new CA certificate was added to your truststore. 
++
+[source,shell]
+----
+keytool -keystore config/http.p12 -list
+----
+When prompted, enter the password for the CA truststore.
++
+The output should contain both the existing CA certificate and your new
+certificate. If you previously used the `elasticsearch-certutil` tool to
+generate your keystore, the alias of the old CA defaults to `http` and the type
+of entry is `PrivateKeyEntry`.
```

Comment:
let's remove this part

Reply:
@jkakavas, just to be clear, you're wanting to remove this part, correct?

> The output should contain both the existing CA certificate and your new
> certificate. If you previously used the `elasticsearch-certutil` tool to
> generate your keystore, the alias of the old CA defaults to `http` and the type
> of entry is `PrivateKeyEntry`.

Reply:
nope, the whole thing, because of https://github.com/elastic/elasticsearch/pull/73781#discussion_r671214300

Reply:
Got it -- I'll remove this section and make the change I proposed in that comment.